projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.spec.ts

```diff
@@ -87,6 +88,13 @@ class MockRoutingService {
   go() {}
 }
 
+class MockIconLoaderService {
+  getSvgPath() {}
+  useSvg() {}
+  getStyleClasses() {}
+  addLinkResource() {}
```

Comment:
Do all these functions need to be mocked?

Reply:
getSvgPath() actually does not need to be mocked. Thanks for pointing it out!